packages/openassistants/openassistants/contrib/sqlalchemy_query.py

```diff
@@ -138,13 +138,13 @@ async def execute_summarization(
             ),
         )
 
-        full = ""
+        full: Union[str, List[Union[str, Dict]]] = ""
 
         async for response_message in deps.summarization_chat_model.astream(
             lc_messages,
             {"tags": ["summarization"]},
         ):
-            full += response_message.content
+            full = merge_content(full, response_message.content)
             yield full
```

Comment:
This yield is still a typing inaccuracy.  The challenge here is that langchain Message.content is *not* a string, pushing that through the rest of the code is gross, and at the end of the day we do need a string.  In the circumstance that it is not a string it is unclear exactly what should happen.  

Result.  This is still a warning, it is correct to be a warning, we should work out what to do.  Input appreciated.  (Personally I think this is bad design on langchain's part)

Reply:
Not sure if we need to support the case where .content isn't a string. Can we write a function that extracts content if it is of type string and otherwise just empty string to guarantee string?